client/client_test.go

```diff
@@ -53,7 +53,7 @@ type clientSuite struct {
 var _ = check.Suite(&clientSuite{})
 
 func (cs *clientSuite) SetUpTest(c *check.C) {
-	cs.cli = client.New()
+	cs.cli = client.New(nil)
```

Comment:
Seems a bit clunky passing `nil` in here but without having a `client.NewWithConfig()` there's little alternative